packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -216,19 +235,21 @@ class ColorScheme with Diagnosticable {
   /// Generate a [ColorScheme] derived from the given `seedColor`.
   ///
   /// Using the seedColor as a starting point, a set of tonal palettes are
-  /// constructed. These tonal palettes are based on the Material 3 Color
-  /// system and provide all the needed colors for a [ColorScheme]. These
-  /// colors are designed to work well together and meet contrast
-  /// requirements for accessibility.
+  /// constructed. These tonal palettes are based on the Material 3 Color system
+  /// and provide all the needed colors for a [ColorScheme]. These colors are
+  /// designed to work well together and meet contrast requirements for accessibility.
   ///
   /// If any of the optional color parameters are non-null they will be
   /// used in place of the generated colors for that field in the resulting
   /// color scheme. This allows apps to override specific colors for their
   /// needs.
   ///
-  /// Given the nature of the algorithm, the seedColor may not wind up as
+  /// Given the nature of the algorithm, the `seedColor` may not wind up as
   /// one of the ColorScheme colors.
   ///
+  /// If the resulting color scheme is too dark, consider setting `variant` to
```

Comment:
Better to explain here that what the variant parameter is for and what its default is. The Variant enum values can explain what they're good for.